holidays/countries/india.py

```diff
@@ -28,269 +37,529 @@ class India(HolidayBase, ChristianHolidays, InternationalHolidays, IslamicHolida
-            2014: (OCT, 23),
-            2015: (NOV, 11),
-            2016: (OCT, 30),
-            2017: (OCT, 19),
-            2018: (NOV, 7),
-            2019: (OCT, 27),
-            2020: (NOV, 14),
-            2021: (NOV, 4),
-            2022: (OCT, 24),
-            2023: (NOV, 12),
-            2024: (NOV, 1),
-            2025: (OCT, 20),
-            2026: (NOV, 8),
-            2027: (OCT, 29),
-            2028: (OCT, 17),
-            2029: (NOV, 5),
-            2030: (OCT, 26),
-        }
-
-        # https://www.timeanddate.com/holidays/india/holi
-        holi_dates = {
-            2001: (MAR, 10),
-            2002: (MAR, 29),
-            2003: (MAR, 18),
-            2004: (MAR, 7),
-            2005: (MAR, 26),
-            2006: (MAR, 15),
-            2007: (MAR, 4),
-            2008: (MAR, 22),
-            2009: (MAR, 11),
-            2010: (MAR, 1),
-            2011: (MAR, 20),
-            2012: (MAR, 8),
-            2013: (MAR, 27),
-            2014: (MAR, 17),
-            2015: (MAR, 6),
-            2016: (MAR, 24),
-            2017: (MAR, 13),
-            2018: (MAR, 2),
-            2019: (MAR, 21),
-            2020: (MAR, 10),
-            2021: (MAR, 29),
-            2022: (MAR, 18),
-            2023: (MAR, 8),
-            2024: (MAR, 25),
-            2025: (MAR, 14),
-            2026: (MAR, 4),
-            2027: (MAR, 22),
-            2028: (MAR, 11),
-            2029: (MAR, 1),
-            2030: (MAR, 20),
-        }
-
-        if self._year in diwali_dates:
-            self._add_holiday("Diwali", diwali_dates[self._year])
-
-        if self._year in holi_dates:
-            self._add_holiday("Holi", holi_dates[self._year])
+        # Diwali.
+        self._add_diwali_india(tr("Diwali"))
+
+        # Holi.
+        self._add_holi(tr("Holi"))
+
+        # Govardhan Puja.
+        self._add_govardhan_puja(tr("Govardhan Puja"))
+
+        # Raksha Bandhan.
+        self._add_raksha_bandhan(tr("Raksha Bandhan"))
+
+        # Janmashtami.
+        self._add_janmashtami(tr("Janmashtami"))
+
+        # Dussehra.
+        self._add_dussehra(tr("Dussehra"))
+
+        # Mahavir Jayanti.
+        self._add_mahavir_jayanti(tr("Mahavir Jayanti"))
+
+        # Maha Shivaratri.
+        self._add_maha_shivaratri(tr("Maha Shivaratri"))
+
+        # Navratri / Sharad Navratri.
+        self._add_sharad_navratri(tr("Navratri / Sharad Navratri"))
+
+        # Ram Navami.
+        self._add_ram_navami(tr("Ram Navami"))
+
+        # Maha Navami.
+        self._add_maha_navami(tr("Maha Navami"))
+
+        # Ganesh Chaturthi.
+        self._add_ganesh_chaturthi(tr("Ganesh Chaturthi"))
+
+        # Makar Sankranti.
+        self._add_makar_sankranti(tr("Makar Sankranti"))
+
+        # Guru Nanak Jayanti.
+        self._add_guru_nanak_jayanti(tr("Guru Nanak Jayanti"))
 
         # Islamic holidays.
-        # Day of Ashura.
-        self._add_ashura_day("Day of Ashura")
+        # Muharram.
```

Comment:
```suggestion
        # Ashura.
```
The established name of this holiday in en_US locale. It is also needed to change the corresponding translation in .po file. See also [this comment](https://github.com/vacanza/holidays/pull/2259#discussion_r1975693758). And [this](https://github.com/vacanza/holidays/pull/2259#discussion_r1976379546), about names' variants.
The same for the rest of Islamic holidays below.